datarobot_provider/example_dags/datarobot_create_execution_environment_dag.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+
+from datetime import datetime
+
+from airflow.decorators import dag
+
+from datarobot_provider.operators.execution_environment import CreateExecutionEnvironmentOperator
+
+
+@dag(
+    schedule=None,
+    start_date=datetime(2022, 1, 1),
+    tags=['example', 'custom model'],
+    params={
+        "execution_environment_name": "Demo Execution Environment",
+        "execution_environment_description": "Demo Execution Environment for Airflow provider",
+        "programming_language": "python",
```

Comment:
Should this be an enum from the datarobot client lib?

Reply:
similar enum exist for custom tasks: CUSTOM_TASK_LANGUAGE = enum(PYTHON="python", R="r", JAVA="java", OTHER="other"), but original "create Execution Environment" method from datarobot client lib didn't refer to this enum, mention just this string can be “python”, “r”, “java” or “other”

Reply:
I wouldn't use the custom tasks enum because they may be similar now but aren't really related so no guarantees PEs will match all values in the future.

Reply:
Also, fwiw, we accept snowflake and azure now too.

Reply:
yes, thats why Im not using enum here, so probably better to keep string value as in original "create Execution Environment" method from datarobot client lib, but datarobot client lib should be refactored first in this case

Reply:
I guess for "snowflake and azure" you are referring to External Prediction Environment for external deployments, but not Custom Execution Environment for custom models

Reply:
Or is it related to "Proxy models"?

Reply:
Ah sorry, I thought this stuff was dealing with Prediction envs not custom models stuff.